packages/components/src/toolbar-button/style.scss

```diff
@@ -25,4 +26,9 @@
 	&:not(:disabled).is-pressed[data-subscript]::after {
 		color: $white;
 	}
+
+	&.is-pressed {
```

Comment:
It seems like the isPressed prop is applied on the "button" prop. I wonder if this style should be moved to the button component instead as it's weird to have a prop that doesn't do anything visually